tests/integration/test_storage_s3_queue/test.py

```diff
@@ -822,7 +823,7 @@ def get_count(node, table_name):
 
 def test_max_set_age(started_cluster):
     node = started_cluster.instances["instance"]
-    table_name = f"max_set_age"
+    table_name = f"max_set_age_{uuid4().hex}"
     dst_table_name = f"{table_name}_dst"
     keeper_path = f"/clickhouse/test_{table_name}"
     files_path = f"{table_name}_data"
```

Comment:
IMHO it is much easier to just have separate path for each test run and not change filenames when we have many of them